src/main/java/mate/controller/CarConnectDriverController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add-driver")
+public class CarConnectDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final String DRIVER_ID = "driverId";
+    private static final String CAR_ID = "id";
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private long carId;
```

Comment:
Andrew was talking about this field , it should be a local var and not common for a whole class, because 2 users can send a request at the same time and 1 of them would be corrupted in that case, because they share this data